scripts/setup-macos.sh

```diff
@@ -35,7 +35,7 @@ PYTHON_VENV=${PYHTON_VENV:-"${SCRIPTDIR}/../.venv"}
 NPROC=$(getconf _NPROCESSORS_ONLN)
 
 DEPENDENCY_DIR=${DEPENDENCY_DIR:-$(pwd)}
-MACOS_VELOX_DEPS="bison boost double-conversion flex fmt gflags glog googletest icu4c libevent libsodium lz4 lzo openssl protobuf@21 simdjson snappy thrift xz xsimd zstd"
+MACOS_VELOX_DEPS="bison boost double-conversion flex gflags glog googletest icu4c libevent libsodium lz4 lzo openssl protobuf@21 simdjson snappy thrift xz xsimd zstd"
```

Comment:
This will not be enough. Problem is that line 39 installs ccache. This brings in fmt11 as well. And since we have the ordering issues will still use fmt11.

Reply:
good point! Let's remove ccache for now and add it back in https://github.com/facebookincubator/velox/pull/10920